index.html

```diff
@@ -128,7 +128,9 @@ <h2>Principles</h2>
   <dd><p>We have a responsibility to build web technologies to counter misinformation, allowing information sources to be traceable and facts to be checkable. The concept of origin and source is core to the web's security model. We will make sure the new web technologies we create do not work against this architectural principle.</p></dd>
 
   <dt>The web must enhance individuals' control and power</dt>
-  <dd><p>The web should empower the people using it, ahead of empowering service and software developers. We will create web technologies and platforms that can't manipulate our users, complicate isolation or encourage addictive behaviors.</p></dd>
+  <dd><p> The web should empower the people using it, ahead of empowering service and software developers. We will create web technologies and platforms th\
```

Comment:
> "We will create web technologies and platforms that can't manipulate our users, complicate isolation or encourage addictive behaviors"

This is a gigantic and implausible promise, as currently phrased. Any information-bearing system designed for human use can be used to manipulate, isolate and addict. Presumably the intent here is something around not making things worse, and to consider negative impacts of technology design, given that the Web is already out there and unlikely to be rescinded?

Reply:
Hi @danbri. How about the following which maybe more correctly captures the sentiment we were going for:  "We recognize that web technologies can be used by service and application developers to manipulate users, complicate isolation and encourage addictive behaviors. We recognize these risks and seek to mitigate against them when creating these technologies and platforms." ?

Reply:
thanks, that is much better

“complicate isolation” is a slightly awkward phrase but I can’t think of an alternative

Reply:
cyberbullying is a thing: https://www.stopbullying.gov/cyberbullying/what-is-it/index.html
and there are various web sites talking about negative effects of the Internet .... (search for 'cyberbullying').

Maybe an external reference would be less awkward and more useful.